mhkit/acoustics/base.py

```diff
@@ -168,9 +255,68 @@ def sound_pressure_spectral_density_level(spsd):
     return out
 
 
+def _validate_method(
+    method: Union[str, Dict[str, Union[float, int]]]
+) -> Tuple[str, Optional[Union[float, int]]]:
```

Comment:
New helper function broken out to avoid pylint "too many locals" that helps validate the method is correctly specified. Used in `band_average` and `time_average`